src/LoginPage/LoginPage.js

```diff
@@ -22,14 +22,16 @@ class LoginPage extends Component {
   submitLogin = (event) => {
     event.preventDefault();
 
-    verifyLogin(this.state.email, this.state.password)
+    return verifyLogin(this.state.email, this.state.password)
       .then((data) => {
         this.setState({ id: data.user.id, userName: data.user.name }, () => {
           this.props.changeUserId(this.state);
-        });
-        this.props.getUsersRatings(data.user.id);
-        this.props.history.push(`/users/${this.state.id}`);
-      })
+				})
+				return data;
+			})
+			.then(async (data) => {
+				await this.props.getUsersRatings(data.user.id);
+				this.props.history.push(`/users/${this.state.id}`)})
```

Comment:
this .then() block is calling on an api call from App. so we need an async await for it to run synchronously + capture the data on the load

Reply:
Such a simple change!!